### ISSUE TITLE
test errors with Numpy 1.10

### ISSUE THREAD

**Issue by msarahan** [Blosc/bcolz#270]
Just a heads-up: I'll try to find time for a PR, unless someone beats me to it.

```
======================================================================
ERROR: test02 (test_ctable.small_iterblocksMemoryTest)
Testing `iterblocks` method with no stop
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\builder\MC_Main\envs\_test\lib\site-packages\bcolz\tests\test_ctable.py", line 1520, in test02
    s += block['f1'].sum(dtype='f16')
  File "C:\Users\builder\MC_Main\envs\_test\lib\site-packages\numpy\core\_methods.py", line 32, in _sum
    return umr_sum(a, axis, dtype, out, keepdims)
TypeError: data type "f16" not understood
```

Many errors like this - "f16" must have been renamed or undefined in Numpy 1.10.

**Comment by msarahan** [comment]
It looks like the right fix is to replace "f16" with a numpy dtype object: http://docs.scipy.org/doc/numpy/user/basics.types.html

Would numpy.longdouble be the best equivalent?  http://stackoverflow.com/questions/9062562/what-is-the-internal-precision-of-numpy-float128

**Comment by FrancescAlted** [comment]
I'd go with longdouble. Thanks!

Sent from my phone

El 20 oct 2015, a les 15:13, Mike Sarahan <notifications@github.com> va escriure:

**Comment by FrancescAlted** [comment]
Fixed in #271.
